frontend/src/models/types.ts

```diff
@@ -0,0 +1,48 @@
+// bad?
+import Word from "./Word";
+
+
+
+// also these & all State/Action can def be moved to a separate file
+/**
+ * Represents types for redux Actions.
+ */
+export enum ActionTypes {
+  REQUEST= "REQUEST",
+  SUCCESS= "SUCCESS",
+  ERROR = "ERROR",
+  CREATE_USER = "CREATEUSER",
+  SIGN_IN = "SIGNIN",
+  AUTHENTICATION_SUCCESS = "AUTHENTICATIONSUCCESS",
+  GET_WORDS = "GETWORDS",
+  GET_WORDS_SUCCESS = "GETWORDSSUCCESS",
+};
+
+
+// i think we should make separate reducers to separate the different states if that makes sense
```

Comment:
I definitely agree here. I don't think we got to a point of figuring out where each reducer lives, but I do fully agree we should split it up.